apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/controller/ItemController.java

```diff
@@ -236,4 +240,15 @@ public ItemDTO get(@PathVariable("appId") String appId,
     return BeanUtils.transform(ItemDTO.class, item);
   }
 
+  @GetMapping(value = "/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/items/page")
```

Comment:
The path `"/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/items/page"` will conflict with the ItemController#get method, so I would suggest using a different path

https://github.com/apolloconfig/apollo/blob/afad056ec7f81dc00eb33fa8094c45ca1915fe8a/apollo-adminservice/src/main/java/com/ctrip/framework/apollo/adminservice/controller/ItemController.java#L227-L230

```suggestion
  @GetMapping(value = "/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/items-with-page")
```

Reply:
Good idea.